pkgs/development/tools/rust/cargo-generate/default.nix

```
@@ -0,0 +1,30 @@
+{ stdenv, fetchFromGitHub, rustPlatform, Security, openssl, pkgconfig }:
+
+rustPlatform.buildRustPackage rec {
+  name = "cargo-generate-${version}";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "ashleygwilliams";
+    repo = "cargo-generate";
+    rev = "v${version}";
+    sha256 = "0n6na6xq4bvs9hc7vc86qqmlrkv824qdmja27b21l2wz3l77r4jb";
+  };
+
+  cargoSha256 = "1y15m2kb2rcrycdrh3ma13kvs23xgcy7zyzy8ap3j1ncd4f99z2i";
```

I got a different hash. If you copied this from another nix expression, it will reuse the hash for the previous artifacts before trying to do a fresh pull.

```suggestion
  cargoSha256 = "00fgzh1s63rr1vs3ahra604m81fc4imx3s09brw2y0n46syhwypi";
```

---

I did copy it and then changed it on `nix-build` error so I assume you're right thanks!